authentication/wildfly/configure-server.cli

```diff
@@ -0,0 +1,5 @@
+embed-server --admin-only=true
+/subsystem=logging/logger=org.wildfly.security:add(level=TRACE)
```

Comment:
This should first check if the logger exists so the server can be reused without being cleaned first.
```
if (outcome = success) of /subsystem=logging/logger=org.wildfly.security:read-resource
    /subsystem=logging/logger=org.wildfly.security:add(level=TRACE)
end-if
```

Reply:
I think that would be a later enhancement, at the moment the shell scrips checks if the server is present and if not installs it and configured is as a single step so this CLI script would not be executed twice against the same server.

Reply:
Ah okay. I missed that. Seems fine then to me.